sdk/src/cameras/ad-fxtof1-ebz/camera_fxtof1.cpp

```diff
@@ -381,7 +382,8 @@ aditof::Status CameraFxTof1::requestFrame(aditof::Frame *frame,
     uint16_t *frameDataLocation;
     frame->getData(FrameDataType::FULL_DATA, &frameDataLocation);
 
-    status = m_depthSensor->getFrame(frameDataLocation);
+    aditof::BufferInfo bufferInfo;
+    status = m_depthSensor->getFrame(frameDataLocation, &bufferInfo);
```

Comment:
We're working on adding the 3D Smart Camera. This should have been added there and not in FXTOF1.

Reply:
Implemented in all cameras.